include/envoy/network/connection.h

```diff
@@ -187,7 +194,10 @@ class Connection : public Event::DeferredDeletable, public FilterManager {
    * Write data to the connection. Will iterate through downstream filters with the buffer if any
    * are installed.
    */
-  virtual void write(Buffer::Instance& data) PURE;
+  virtual void write(Buffer::Instance& data, bool last_byte) PURE;
```

Comment:
nit: I might consider actually calling this `end_stream` vs. `last_byte` like we do up in HTTP land, just to match the concepts. But I admit this is bikeshedding and I don't feel strongly about it.

Reply:
I'll change it to keep things consistent.